rosidl_generator_c/msg/DynamicArrayPrimitivesNested.msg

```diff
@@ -0,0 +1 @@
+DynamicArrayPrimitives[] msgs
```

Comment:
Given that it's not code I think we don't care. But feel free to commit a modified version with the newline if you feel like it